projects/plugins/jetpack/extensions/blocks/simple-payments/simple-payments.php

```diff
@@ -40,6 +41,8 @@ function register_block() {
  * @return string
  */
 function render_block( $attr, $content ) {
+	Jetpack_Gutenberg::load_styles_as_required( FEATURE_NAME );
```

Comment:
I don't think we need this and the new `view.scss` file, since `projects/plugins/jetpack/modules/simple-payments/simple-payments.css` gets enqueued for the block as well, here:
https://github.com/Automattic/jetpack/blob/2bce3be14f46eee0120a7c115ed439306cee93c7/projects/plugins/jetpack/extensions/blocks/simple-payments/simple-payments.php#L57

In fact, I don't think the css is generated from that `view.scss` file right now; I believe you would need to do something like this for it to be included:
```diff
diff --git a/projects/plugins/jetpack/extensions/blocks/simple-payments/index.js b/projects/plugins/jetpack/extensions/blocks/simple-payments/index.js
index e1589a5ee..8fa8c5a71 100644
--- a/projects/plugins/jetpack/extensions/blocks/simple-payments/index.js
+++ b/projects/plugins/jetpack/extensions/blocks/simple-payments/index.js
@@ -25,6 +25,7 @@ import simplePaymentsExample1 from './simple-payments_example-1.jpg';
  * Styles
  */
 import './editor.scss';
+import './view.scss';
 
 export const name = 'simple-payments';
 
diff --git a/projects/plugins/jetpack/extensions/blocks/simple-payments/view.js b/projects/plugins/jetpack/extensions/blocks/simple-payments/view.js
new file mode 100644
index 000000000..741c50d6a
--- /dev/null
+++ b/projects/plugins/jetpack/extensions/blocks/simple-payments/view.js
@@ -0,0 +1,4 @@
+/**
+ * Internal dependencies
+ */
+import './view.scss';
```

Reply:
Thanks for this! The `view.scss` was working for me, but I only added it because the CSS was not working on wpcom for me without it. Now that I dig in deeper, I finally figured out there was an issue with our syncing that was causing the `module` code to sync to the wrong location on the sandbox. Now that's resolved, my changes to `simple-payments.css` are being reflected in my wpcom tests as well, and I don't need the `view.css` workaround 🎉